docs/use_case_3-1.md

```diff
@@ -90,3 +90,19 @@ As for the previous Use Cases 1 and 2, the first step is to collect all the inpu
 - For Step 1 (Fever Patient Sequences collection), select `patient collection` (this should be already selected).
 - For Step 2, select the `nucleotide vir2 blast database` (this should also be already selected)
 - As usual, check the box `Send results to a new history`, edit the name of the new history to `History for Use Case 3-1`, and `Execute` the workflow ! Note, that for complex workflows with dataset collections in input, the actual warning that the workflow is started may take time to show up.
+
+---
+
+## Results
+
+As you can see the results for this use case differ whether you use [Metavisitor](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0168397) or Metavisitor2.
+There are failed (red) datasets in this history, this is normal, it means not all patients had sequences matching the viral database. You will notice that only `patient 0629-453` has contigs matching HIV sequences, this is a false positive, you can tell it's a false positive by looking at the other matching sequences names which are very heterogeneous (bovine, shrimp and plant viruses). To make sure its a false positive you can:
```

Comment:
this is normal --> this is expected

Reply:
> you can tell it's a false positive by looking at the other matching sequences names which are very heterogeneous (bovine, shrimp and plant viruses)

Je ne comprends pas l'argument

Reply:
je veux dire par là que le contig hit des sequences qui sont trés différentes. Virus de crevette, vache, plante...
Je veux donc faire entendre à l'utilisateur que le fait de matcher des séquences très différentes peut être un signe de faux positif.

Reply:
Tu as rephrasé l'argument mais cela ne change pas ma question/remarque:

Pourquoi  le fait de matcher des séquences très différentes peut être un signe de faux positif ?
Note que l'on ne peut pas matcher des séquences très différentes dans un blast, je pense que tu veux dire `matcher des séquences d'origines très différentes`... et ma clarification ne te dispense pas de répondre à ma question :-) : 

Pourquoi  le fait de matcher des séquences _d'origines_ très différentes peut être un signe de faux positif ?

Par ailleurs si X peut être signe de Y, alors X peut aussi _ne pas_ être signe de Y. L'information est marginale si elle est présentée comme cela. Peut on faire intéressant ?

Reply:
Oui je voulais dire matcher des séquences d'origines très différentes.

Si on essaye de détecter du VIH dans des séquençages d'humains, le fait de matcher des séquences d'origines différentes, montre un manque de spécificité. Sur tout si c'est la même région du contig qui match les différentes séquences.
Bien évidemment ça ne veux pas dire que c'est forcément un faux positif, mais qu'il faut faire plius de tests pour clarifier la situation.